### PR TITLE
fixed https://github.com/NuGet/Home/issues/1514

### PR DESCRIPTION
**Pull request by zhili1208**
https://github.com/NuGet/Home/issues/1514
port
https://nuget.codeplex.com/SourceControl/network/forks/StephenCleary/Issue4013/changeset/2803f4895ea678d0b0cfbb5a5e2961ec0dec40f7

### CODE REVIEW

**Comment by sliekens** [comment]
bump

**Comment by yishaigalatzer** [comment]
@zhili1208 where are the tests covering this scenario? Please hold off on merging until we open 2.10 for commits

**Comment by sliekens** [comment]
See this repro:
https://github.com/StevenLiekens/MyPortableLib

This fiddle describes the fix:
https://dotnetfiddle.net/VB8mta

**Comment by sliekens** [comment]
Do you want to hold off merging because you expect this to break other scenarios? I don't see how.

**Comment by yishaigalatzer** [comment]
Because
1. It introduced no tests, and we have to add then
2. Because we are in a process of moving out engineering system and we want to merge slowly and see that everything is passing.

This should be in by end of week

**Comment by MeniZalzman** [comment]
:watch: @zhili1208 can you please add test for this change?

**Comment by zhili1208** [comment]
yes, I will do it later

File: test/EndToEnd/ProjectTemplates/PortableLib.zip/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Resources;
```
Comment (yishaigalatzer) [inline test/EndToEnd/ProjectTemplates/PortableLib.zip/Properties/AssemblyInfo.cs:1]: this is probably not necessary

**Comment by yishaigalatzer** [comment]
:shipit:

**Pull request closed without merging**